Production/govuk_ios/Model/DeviceInformation.swift

```diff
@@ -1,7 +1,17 @@
 import Foundation
+import UIKit
 
-struct DeviceModel: CustomStringConvertible {
-    var description: String {
+protocol DeviceInformationInterface {
```

Comment:
Worth aligning these names with `AppVersionProvider`? They are essentially the same thing, just one is for the app, one is for the device, but they are named inconsistently

Reply:
Good point, renamed 👍